.readthedocs.yml

```
@@ -0,0 +1,21 @@
+version: 2
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.10"
```

Would recommend using 3.11 and up as it is faster in general.
```suggestion
    python: "3.11"
```